appengine-java11/appengine-simple-jetty-main/README.md

```diff
@@ -70,7 +70,7 @@ the provided server is running your application as expected.
 
 - Start the server with your `WAR` file as an argument:
 ```
-mvn exec:java -Dexec.args="<path/to/your/war/file"
+mvn exec:java -Dexec.args="path/to/your/war/file"
```

Comment:
`-Dexec.args="target/path.war"` perhaps or something that mentions target

Reply:
done.